sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1620,7 +1623,11 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
   }
 
   def get[T](entry: ConfEntry[T]): T = {
-    entry.get(conf)
+    // 'converter' is wrapped with 'checkValue' (if the config item has), so we can check the
```

Comment:
Do not do this here, let the `entry` take care of the converting things in its `get`.

Reply:
Fixed.